styles.css

```diff
@@ -0,0 +1,94 @@
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url("/images/header-shapes-mobile.png");
```

Comment:
- [ ] The background image is for the headline section, not the entire body, so kindly fix it.